composer.json

```diff
@@ -26,7 +26,8 @@
     },
     "require-dev": {
         "php-http/guzzle7-adapter": "^1.0",
-        "phpunit/phpunit": "^8.0|^9.0"
+        "phpunit/phpunit": "^8.0|^9.0",
+        "php-http/message-factory": "^1.1"
```

Comment:
@ekyna this requirement as been added to wait for a proper fix from Payum since this packages is deprecated